dapps/profile.vue/src/components/wallet/buy/BuyEve.ts

```diff
@@ -107,6 +108,12 @@ export default class BuyEveComponent extends mixins(EvanComponent) {
     success: false,
   };
 
+  @Watch('contactForm.country.value')
+  onCountryChange(country) {
+    // change tax value depends on country select
+    this.taxValue = country === 'DE' && !this.contactForm.vat.value ? 19 : 0;
```

Comment:
This looks like it's possible to buy from Germany without paying VAT. IMO this shouldn't be the behaviour.

Reply:
see also: https://evannetwork.atlassian.net/browse/CORE-395